content/05.discussion.md

```diff
@@ -78,6 +77,13 @@ For example, a person may not be mentioned or quoted in the article because of l
 A more accurate reflection of journalists' sources would be a self-maintained record of people they interview.
 Our work examines disparities with respect to recognition within articles, which can be measured by mentions, quotes, or citations of people.
 
+Another major limitation of our study, is that we only used articles published by _Nature_ or _Springer Nature_ as a comparator.
+Not all papers are interesting to the general public and likely to be covered by journalists.
```

Comment:
One reason to focus on just _Nature_ is that one could make the argument that these are perhaps more likely to be of general/broad interest?

Reply:
Yes, that's possible, but I'm not sure how much justification we need for using Nature/Springer Nature. I think choosing Nature alone as one kind of vignette of this problem is enough. 

I could add more justification if you think it's warranted.